### PR TITLE
Event tests validations

### PR DESCRIPTION
**Pull request by martom87**

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -1,8 +1,16 @@
 class Event < ApplicationRecord
+
+  validates :title, :capacity, :ended_at, :started_at, presence: true
+  validates :description, length: {minimum: 10, maximum: 500}, presence: true
+  validates :capacity, numericality: {only_integer: true}
```
Comment (jluczak) [inline app/models/event.rb]: Capacity must be higher than 1

File: db/schema.rb [hunk 2]
```diff
@@ -47,6 +47,15 @@
...
 
+  create_table "places", force: :cascade do |t|
+    t.string "address"
+    t.float "latitude"
+    t.float "longitude"
+    t.string "contact"
+    t.date "day"
+    t.time "hour"
+  end
+
```
Comment (jluczak) [inline db/schema.rb]: delete changes in db/schema

File: spec/models/comment_spec.rb [hunk 3]
```diff
@@ -2,4 +2,4 @@
 
 RSpec.describe Comment, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
-end
+git end
```
Comment (mpiask) [inline spec/models/comment_spec.rb]: ☝️ ??

File: spec/models/event_spec.rb [hunk 4]
```diff
@@ -1,5 +1,45 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (mpiask) [inline spec/models/event_spec.rb]: ☝️ ???

**Review by jluczak — APPROVED** [review]

**Pull request merged**